File: programme/area-of-circle/area-of-circle.php
<?php
// Write a programme to find the area of circle

function calculate_area(int | float $r)
{

    if (!is_numeric($r)) {

        return 'Invalid input: the input is not a number.';
    } else {
        return pi() * $r * $r;
    }
}

$r = readline("Enter radius : ");

$area = round(calculate_area($r), 2);
print "\nRadius : {$r}";
print "\nArea   : {$area}\n";

File: programme/area-of-circle/area-of-circle.cpp
// Write a programme to find the area of circle
#include <bits/stdc++.h>

using namespace std;

#define pi 3.141

int main()
{
    int r;

    // input radius
    cout << "Enter radius: ";
    cin >> r;

    // output area
    cout << "Area: " << floor((pi * r * r) * 100) / 100;
}
